openpype/plugins/publish/extract_thumbnail.py

```diff
@@ -445,7 +445,11 @@ def _create_frame_from_video(self, video_file_path, output_dir):
         # Set video input attributes
         max_int = str(2147483647)
         video_data = get_ffprobe_data(video_file_path, logger=self.log)
-        duration = float(video_data["format"]["duration"])
+        duration = max(
```

Comment:
```suggestion
        # Use duration of the individual streams since it is returned with
        # higher decimal precision than 'format.duration'. We need this
        # more precise value for calculating the correct amount of frames
        # for higher FPS ranges or decimal ranges, e.g. 29.97 FPS 
        duration = max(
```